docs/eslint-angular/rules/use-none-component-view-encapsulation.md

```diff
@@ -7,11 +7,12 @@ Last reviewed: 2023-05-23
 
 # [Use none component view encapsulation](../../../lib/eslint-angular/src/rules/use-none-component-view-encapsulation/use-none-component-view-encapsulation.ts "Defined in use-none-component-view-encapsulation.ts")
 
-Custom ESLint rule which check if component uses ViewEncapsulation.None. It has been implemented because None encapsulation makes themes styling easier.
-It also allows to autofix.
+Custom ESLint rule that requires components to use ViewEncapsulation.None. It has been implemented because None encapsulation makes themes styling easier.
```

Comment:
This is incorrect rebase as it belongs to another PR

Reply:
rebased the branch - APSS-2135-replace-moment-to-date-fns now only my changes related to moment to date-fns conversion are showing.